dspace-api/src/main/java/org/dspace/content/dao/RelationshipTypeDAO.java

```diff
@@ -36,5 +37,6 @@
     RelationshipType findbyTypesAndLabels(Context context,
                                           EntityType leftType,EntityType rightType,String leftLabel,String rightLabel)
                                                 throws SQLException;
+    List<RelationshipType> findByLeftOrRightLabel(Context context, String label) throws SQLException;
```

Comment:
Please add Javadocs for this new public interface method.

Reply:
The Javadocs have been included